CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
+- Fix GCC 12 compiler warnings
```

Comment:
Please remove this entry because it is not interesting end users.

Reply:
I removed it.